internal/definition/validate_test.go

```diff
@@ -101,7 +108,8 @@ func Test_ValidatePipeline(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			def_file = tt.defFunc
-			output, err := ValidateDefinition(ctx, tt.fpath, []source.PolicySource{})
+			pathExists = tt.exists
```

Comment:
The nice thing about using afero, is that you can load an in-memory filesystem. Rather than mocking pathExists, you can change the test to actually create the file, or not. That's what most of the other tests do. It's the reason we use `utils.FS(ctx)`.

Reply:
Replaced OS commands with Afero. Very neat!